dts/arm/yaml/st,stm32-usb.yaml

```diff
@@ -0,0 +1,56 @@
+---
+title: STM32 USB
+id: st,stm32-usb
+version: 0.1
+
+description: >
+    This binding gives a base representation of the STM32 USB controller
+
+inherits:
+  - !include usb.yaml
+
+properties:
+  - compatible:
+      type: string
```

Comment:
I wonder if this would be possible to put "compatible" property in usb.yaml.
I guess it is required for all usb nodes.
We would only keep compatible/constraint defined here

Reply:
I tested it and it works.